askama_derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "askama_derive"
-version = "0.11.2"
+version = "0.12.0"
```

Comment:
I un-exported a bunch of symbols. Older `askama_shared` versions would break with this update. Every item in https://github.com/djc/askama/blob/main/askama_derive/src/lib.rs#L8-L10 is gone after the PR.

Reply:
I removed the features section, because they are now handles in askama_shared.

I am not 100% sure, but I guess if I re-added the features, incrementing the patch version only should be fine.